### ISSUE TITLE
Using google tag manger with react-ga

### ISSUE THREAD

**Issue by hhsadiq** [react-ga/react-ga#347]
Can we use react-ga and Tag Manager together? If not, is there a workaround to get it working?

**Comment by danbo** [comment]
https://medium.com/growth-analytics/migrating-your-on-page-tracking-from-analytics-js-to-the-new-gtag-js-part-1-bbb4d0a29d39
https://developers.google.com/analytics/devguides/collection/gtagjs/migration

**Comment by Franatec** [comment]
Yes. This would be great. Search console no longer validates with ga.js, gtag.js is required.

**Comment by aprilandjan** [comment]
We have a paid analytic account and it must be used with google tag manager. The `react-ga` provides no option to switch to `gtag`. Seems the author doesn't have any to plan of this.

**Comment by fabienpinel** [comment]
Same here. How did you guys do ?

**Comment by MostafaaRamadan** [comment]
Any updates?
